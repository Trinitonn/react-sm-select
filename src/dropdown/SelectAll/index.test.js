import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { SelectAll } from './index';

describe('SelectAll', () => {
  it('should render properly', () => {
    const wrapper = shallow(
      <SelectAll
        option={{ label: 'One', value: 'one' }}
        checked={true}
        focused={true}
        onClick={() => {}}
      />
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
