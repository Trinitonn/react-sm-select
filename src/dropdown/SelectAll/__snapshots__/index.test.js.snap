// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectAll should render properly 1`] = `
<li
  className="OptionList__item"
>
  <Option
    Option={[Function]}
    checked={true}
    focused={true}
    onClick={[Function]}
    option={
      Object {
        "label": undefined,
        "value": "",
      }
    }
  />
</li>
`;
